qa-anvil.planx-pla.net/portal/gitops.json

```diff
@@ -240,34 +237,11 @@
       "dataType": "subject",
       "nodeCountTitle": "Subjects",
       "fieldMapping": [
-        { "field": "sex", "name": "Sex" },
```

Comment:
these are redundent field mappings, guppy will do this kind of conversion by default